components/member-card/card.module.scss

```diff
@@ -79,6 +79,58 @@
   font-size: 0.88rem;
 }
 
+.walletContainer {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  align-items: center;
+  border-radius: 2px;
+  width: 150px;
+  height: 70px;
+  padding: 10px 5px 2px;
+  margin-bottom: 8px;
+  margin-left: 6px;
+  background-color: #EEEEEE;
+
+  .coinData {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .coinData>p {
+    height: 2px;
+  }
+
+  .coinData>p:first-child {
+    font-size: 12px;
+    font-weight: 500;
+    transform: translateY(-25px);
+    margin-right: 8px;
+  }
+
+  .coinData>p:last-child {
```

Comment:
let's not apply CSS in this way, not a fan of it. Cant we give the last child any specific class name and then apply the css?